conf/livy-env.sh.template

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
```

Comment:
Could you add a line end at the end of this file?

Also, could you call it `livy-env.sh` (without the `.template`)? There's no reason for it to be a template, since it's a no-op in its current state.